src/provisioner/const.py

```diff
@@ -20,3 +20,4 @@
 DEFAULT_LOG_LEVEL = "INFO"
 SUPPORTED_LOG_LEVELS = ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
 RELEASE_INFO_URL = "yaml:///opt/seagate/cortx/RELEASE.INFO"
+DISRUPTIVE_UPGRADE = False
```

Comment:
Please name disruptive as "cold upgrade". Its small and more meaningful. 

COLD_UPGRADE_KEY = 'COLD_UPGRADE'
COLD_UPGRADE_MODE = 'False'

Reply:
@ujjwalpl 
Instead can we have something like this:
SUPPORTED_UPGRADE_MODES = ["ROLLING", "COLD"]
DEFAULT_UPGRADE_MODE = "ROLLING"

Reply:
We should not avoid explicit concept for rolling upgrade. Rolling upgrade is just a method. By default upgrade is supposed to be rolling upgrade which will be run 99.99%. COLD is extreme case, enabled through an option.

Reply:
@ujjwalpl This is internal code logic, not exposed to user... By default it will always be rolling upgrade, with explicit option to execute disruptive upgrade using "cold" keyword...

Reply:
Lets keep these as below (instead of adding special meaning to boolean values):
`SUPPORTED_UPGRADE_MODES = ["ROLLING", "COLD"]`
`DEFAULT_UPGRADE_MODE = "ROLLING"`

Reply:
UPGRADE_MODE_KEY = "UPGRADE_MODE"
UPGRADE_MODE = ""

done